### DISCUSSION TITLE
Request to Reconsider the Deletion of keycloak-ui Repository

### DISCUSSION THREAD

**Discussion by fredericoschardong** [keycloak/keycloak#30411] [Ideas]
Hello Keycloak Team,

I hope this message finds you well. I am writing to express a concern regarding the recent deletion of the keycloak-ui repository. My team and I have been relying on this repository for integrating and managing authentication services in our projects. The sudden removal has posed significant challenges to our ongoing deployments.

We understand that projects evolve and certain repositories may no longer be actively maintained. However, we kindly request that instead of deleting the repository, it could be marked as archived. This would still prevent new issues or pull requests while allowing teams like ours to access the existing codebase and documentation, which are invaluable for our operations.

Archiving the repository would offer a compromise that maintains the integrity and history of the project without requiring ongoing maintenance from your team. It would greatly benefit the community and developers who have built solutions around this repository.

**Comment by Bredstone** [comment]
For those who might need it, [here is an unmodified fork of Keycloak-UI repository](https://github.com/Bredstone/Keycloak-UI) :)

**Comment by jonkoops** [comment]
Sorry to hear you have been inconvenienced, that is never our intention. Exactly how is your team dependent on our outdated code? No changes have been committed for over a year, so if your team had previously forked this code there should be no issue.

**Reply by fredericoschardong** [reply]
Perhaps I should have forked it back in the day to avoid this hustle. We made some changes to the admin interface of version 20.0.2, which we simply applied on top of a shallow clone of the official repo's 20.0.2 tag then compiled it. All this happened inside a Dockerfile. Since we have our own gitlab instance, we figured this was the best approach.

Our immediate solution is to use Brendon's fork (he is on our team).

**Reply by jonkoops** [reply]
Probably best to use the fork, but I would recommend to start building against the main repo theme at some point if you want it to continue to work on newer versions in the future.
